tools/dependencies/known-oap-backend-dependencies.txt

```diff
@@ -12,6 +12,7 @@ armeria-graphql-1.14.1.jar
 armeria-graphql-protocol-1.14.1.jar
 armeria-protobuf-1.14.1.jar
 audience-annotations-0.5.0.jar
+banyandb-java-client-0.1.0-SNAPSHOT.jar
```

Comment:
FYI @apache/skywalking-committers This is a temp adding. Once e2e tests are added, we would be confident to release the BanyanDB server and its Java client 0.1.0. Then we could add this back.

@lujiajing1126 I would like to merge this, and you could add e2e tests later through new PR(s). Is that good to you?

Reply:
Make sense to me!